mautic/mautic
# Fix graph social monitoring

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | X
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #4186
| BC breaks? | 
| Deprecations? | 

[//]: # ( Required: )
#### Description:
The graph in the social monitoring is wrong because it counts the number of rows in the `monitor_post_count `table rather than adding up the `post_count` column 


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a social monitoring
2. Get some lead in monitoring
3. See the graph with only one "data" per day

#### Steps to test this PR:
1. Apply this PR
2. See the graph with the number of all "data" per day
